### ISSUE TITLE
VoiceAttack, Start Gear Announcement, setting SRA_loop false still loops

### ISSUE THREAD

**Issue by samkatakouzinosracing** [SIMRacingApps/SIMRacingApps#136]
I didn't want to hear gear changes through SRA.
I set the boolean to false.
Gear changes were still announced.

I changed logic to set and test for integers (0,1).
This works better.

**Comment by jfgilliam** [comment]
You can also remove it from the Start Spotter command.

**Comment by samkatakouzinosracing** [comment]
TL:DR; Please change global SRA_Loop to command scope ~SRA_Loop for  "Start Flags", "Start Gear Announcement", "Start Leader Change", "Start Pit Road".

Ok, page 158 of VA manual talks about "Advanced Variable Control (Scope)".
Variables can be declared globally (as you are doing), profile scoped and command scoped. 
Global variables (variables available to all commands and profiles) have no prefix.
Profile scoped commands are prefixed by ">" and are available to all commands within the profile but not other profiles.
Command scoped commands are prefixed by "~" and are available to all command entries and sub-commands. They are not accessable to other commands. 
So! I changed all SRA_Loop declarations and references to ~SRA_Loop in "Start Flags", "Start Gear Announcement", "Start Leader Change", "Start Pit Road".
Changing to Command scope variables now works as I intended.

**Comment by jfgilliam** [comment]
Probably should do this to all my variables, but that's a lot of work. I name most of the uniquely to prevent this problem. I don't think VA had local variables when I first built this profile, but I don't remember.

**Comment by samkatakouzinosracing** [comment]
Thank you.
